beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -4,7 +4,6 @@ import (
 	"strconv"
```

Comment:
What should go in the shared/structs.go file vs the one next to the handlers? aren't all the types shared? 🤔

Reply:
requests/responses should go next to the handlers because they are 1-to-1 with endpoints